### PR TITLE
Viite 1023 Suravage links included in the initial road address download

### PR DESCRIPTION
**Pull request by skosonen**
Change the import road address batch to get also the suravage VVH-links.
Change the update road address geometry batch to handle the suravage links as well.

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala [hunk 1]
```diff
@@ -1170,6 +1170,10 @@ class VVHSuravageClient(vvhRestApiEndPoint: String) extends VVHRoadLinkClient(vv
   }
 
   def fetchSuravageByLinkIdsF(linkIds: Set[Long] = Set()): Future[Seq[VVHRoadlink]] = {
-    Future(queryByLinkIds(linkIds, None, true, extractRoadLinkFeature, withLinkIdFilter))
+    Future(fetchSuravageByLinkIds(linkIds))
+  }
+
+  def fetchSuravageByLinkIds(linkIds: Set[Long] = Set()): Seq[VVHRoadlink] = {
+    queryByLinkIds(linkIds, None, true, extractRoadLinkFeature, withLinkIdFilter)
```
Comment (Campanha) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala]: I've think better about this. I realized that in lrmListWithLinkSources we are using the linkSource from roadLinks which are all being build by extractRoadLinkFeature() when queryByLinkIds() that gives LinkGeomSource.NormalLinkInterface by default.

Should we modify the linkSource within each diferent vvhEndPoint class? 
Making something like 
*queryByLinkIds(linkIds, None, true, extractRoadLinkFeature, withLinkIdFilter).map(_.copy(linkSource = linkGeomSource))*

Comment (Campanha) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala]: if we use this approach we need to probably define a new overriden fetchByLinkIds also for the VVHComplementaryClient

Comment (oappicgi) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala]: linkgeomsource should ba saved in VVHClient when we fetch roadlinks

Comment (oappicgi) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala]: for example for suravage it is set in (VVHClient) row 1161 and for complimentary 1025.

Comment (skosonen) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala]: In VVHClient.scala:633 we set linkGeomSource, which is defined in each VVHRoadLinkClient implementation. For example VVHSuravageClient has LinkGeomSource.SuravageLinkInterface.

**Pull request merged**